tiatoolbox/models/dataset/abc.py

```diff
@@ -33,14 +33,17 @@ class PatchDatasetABC(ABC, torch.utils.data.Dataset):
     """Defines abstract base class for patch dataset.
 
     Attributes:
-        return_labels (bool, False): `__getitem__` will return both the img and
-          its label. If `labels` is `None`, `None` is returned
-        preproc_func: Preprocessing function used to transform the input data. If
-          supplied, then torch.Compose will be used on the input preprocs.
-          preprocs is a list of torchvision transforms for preprocessing the
-          image. The transforms will be applied in the order that they are given in
-          the list. For more information, use the following link:
-          https://pytorch.org/vision/stable/transforms.html.
+        return_labels (bool, False):
```

Comment:
cannot find this property

Reply:
@simongraham  Can you clarify this please? I see you added this code.

Reply:
I think we can remove the property for now. We may want to add something similar when we incorporate training, but for now we can get rid of it.